src/store/configureStore.js

```diff
@@ -8,40 +8,35 @@ import {
   getTracker,
   createTrackerMiddleware
 } from 'cozy-ui/transpiled/react/helpers/tracker'
-//import { isReporterEnabled, getReporterMiddleware } from 'lib/sentry'
 
 import appReducers from '../reducers'
 
 const configureStore = (client, t, persistedState) => {
   // Enable Redux dev tools
   const composeEnhancers =
     (__DEVELOPMENT__ && window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__) || compose
-  const cozyReducer = client.reducer()
-  // reducers
-  //const reducers = [appReducers, persistedState, cozyReducer]
-  const reducers = combineReducers({
-    appReducers,
-    persistedState,
-    cozy: cozyReducer
-  })
+
   // middlewares
   const middlewares = [thunkMiddleware.withExtraArgument({ client, t })]
 
+  // tracker middleware
   if (shouldEnableTracking() && getTracker()) {
     middlewares.push(createTrackerMiddleware())
   }
+
+  // logger middleware
   if (flag('logs') && __DEVELOPMENT__) {
-    // eslint-disable-line
     // must be the last middleware in chain https://git.io/vHQpt
     const loggerMiddleware = createLogger()
     middlewares.push(loggerMiddleware)
   }
-  /* if (isReporterEnabled()) {
-    middlewares.push(getReporterMiddleware(cozyClient))
-  } */
 
   const store = createStore(
-    reducers,
+    combineReducers({
+      appReducers,
+      cozy: client.reducer()
+    }),
+    persistedState,
```

Comment:
why `persistedState` is not in `combineReducers` anymore? We're changing the behavior by doing that. It should not be tagged as refactor if this is the case

Reply:
Damned my bad, it's a copy/past from Banks